template/templates/views/entry.html

```diff
@@ -158,21 +191,27 @@ <h1 dir="auto">
             <div class="entry-enclosure">
                 {{ if hasPrefix .MimeType "audio/" }}
                     <div class="enclosure-audio">
-                        <audio controls preload="metadata">
+                        <audio controls preload="metadata"
+                               data-last-position="{{ .MediaProgression }}"
+                               data-save-url="{{ route "saveEnclosureProgression" "enclosureID" .ID }}"
+                        >
 				{{ if (and $.user (mustBeProxyfied "audio")) }}
-				    <source src="{{ proxyURL .URL }}" type="{{ .MimeType }}">
+				    <source src="{{ proxyURL .URL }}" type="{{ .Html5MimeType }}">
 				{{ else }}
-				    <source src="{{ .URL | safeURL }}" type="{{ .MimeType }}">
+				    <source src="{{ .URL | safeURL }}" type="{{ .Html5MimeType }}">
```

Comment:
There is no substitution of MimeType for audio right now, but I feel it's better to use this field on every situations its name suggest.  I did not used it for image because MimeType is not used at all for the Browser.